packages/genesys-spark-components/src/components/beta/gux-side-panel/example.html

```diff
@@ -0,0 +1,143 @@
+          >
+        </gux-cta-group>
+      </div>
+    </gux-modal-side-panel-beta>
+
+    <gux-cta-group>
+      <gux-button slot="secondary" id="modal-small">Show small</gux-button>
+      <gux-button slot="secondary" id="modal-medium">Show medium</gux-button>
+      <gux-button slot="secondary" id="modal-large">Show large</gux-button>
+    </gux-cta-group>
+  </div>
+
+  <gux-side-panel-beta size="medium" id="dynamic-example">
+    <gux-side-panel-heading slot="heading" level="3" icon="fa/diamond-regular">
+      Side panel title
+    </gux-side-panel-heading>
+    <h2 slot="description">Pellentesque habitant morbi tristique sen</h2>
+    <div slot="content">
+      Pellentesque habitant morbi tristique senectus et netus et malesuada fames
+      ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget,
+      tempor sit amet, ante. Donec eu libero sit amet quam egestas semper.
+      Aenean ultricies mi vitae est. Mauris placerat eleifend leo.
+    </div>
+    <div slot="footer">
+      <gux-cta-group>
+        <gux-button slot="primary" onclick="notify(event)">Primary</gux-button>
+        <gux-button slot="secondary" onclick="notify(event)"
+          >Secondary</gux-button
+        >
+        <gux-button
+          slot="dismiss"
+          onclick="document.getElementById('dynamic-example').style.display = 'none'"
+          >Dismiss</gux-button
+        >
+      </gux-cta-group>
+    </div>
+  </gux-side-panel-beta>
+</div>
+
+<style
+  onload="(function () {
+  const modalSidePanel = document.getElementById('modal-example');
+  const dynamicSidePanel = document.getElementById('dynamic-example');
+  
+  document.getElementById('modal-small').addEventListener('click', function () {
+    modalSidePanel.setAttribute('size', 'small');
+    modalSidePanel.showModal();
+  });
+
+  document.getElementById('modal-medium').addEventListener('click', function () {
+    modalSidePanel.setAttribute('size', 'medium');
+    modalSidePanel.showModal();
+  });
+
+  document.getElementById('modal-large').addEventListener('click', function () {
+    modalSidePanel.setAttribute('size', 'large');
+    modalSidePanel.showModal();
+  });
+
+  document.getElementById('toggle-small').addEventListener('click', function () {
+    dynamicSidePanel.size = 'small';
+    dynamicSidePanel.style.display = 'block';
+  });
+
+  document.getElementById('toggle-medium').addEventListener('click', function () {
+    dynamicSidePanel.size = 'medium';
+    dynamicSidePanel.style.display = 'block';
+  });
+
+  dynamicSidePanel.addEventListener('sidePanelDismiss', event => {
+    dynamicSidePanel.style.display = dynamicSidePanel.style.display === 'none' ? 'block' : 'none';
+  });
+  modalSidePanel.addEventListener('modalSidePanelDismiss', event => notify(event));
+})()"
+>
+  .preview {
+    padding: 0 !important;
+  }
+
+  #side-panel-wrapper {
+    display: flex;
+    height: 100%;
+  }
+
+  #content {
+    flex: 1 1 auto;
+    padding: 27px 30px;
+  }
```

Comment:
I'm getting a horizontal scrollbar on Chrome when I look at the examples. Not sure what's causing it. 
![Screenshot 2024-11-05 at 09 57 32](https://github.com/user-attachments/assets/38ab3b8b-9bcb-42d6-878c-c554ca2059cd)